SDWebImage/UIButton+WebCache.m

```diff
@@ -129,32 +137,36 @@ - (void)sd_setBackgroundImageWithURL:(nullable NSURL *)url
                              options:(SDWebImageOptions)options
                            completed:(nullable SDExternalCompletionBlock)completedBlock {
     if (!url) {
-        [self.imageURLStorage removeObjectForKey:backgroundImageURLKeyForState(state)];
+        [self.sd_imageURLStorage removeObjectForKey:backgroundImageURLKeyForState(state)];
     } else {
-        self.imageURLStorage[backgroundImageURLKeyForState(state)] = url;
+        self.sd_imageURLStorage[backgroundImageURLKeyForState(state)] = url;
     }
     
     __weak typeof(self)weakSelf = self;
     [self sd_internalSetImageWithURL:url
                     placeholderImage:placeholder
                              options:options
-                        operationKey:[NSString stringWithFormat:@"UIButtonBackgroundImageOperation%@", @(state)]
+                        operationKey:backgroundImageOperationKeyForState(state)
                        setImageBlock:^(UIImage *image, NSData *imageData) {
                            [weakSelf setBackgroundImage:image forState:state];
                        }
                             progress:nil
                            completed:completedBlock];
 }
 
+#pragma mark - Cancel
+
 - (void)sd_cancelImageLoadForState:(UIControlState)state {
-    [self sd_cancelImageLoadOperationWithKey:[NSString stringWithFormat:@"UIButtonImageOperation%@", @(state)]];
+    [self sd_cancelImageLoadOperationWithKey:imageOperationKeyForState(state)];
 }
 
 - (void)sd_cancelBackgroundImageLoadForState:(UIControlState)state {
-    [self sd_cancelImageLoadOperationWithKey:[NSString stringWithFormat:@"UIButtonBackgroundImageOperation%@", @(state)]];
+    [self sd_cancelImageLoadOperationWithKey:backgroundImageOperationKeyForState(state)];
 }
 
-- (SDStateImageURLDictionary *)imageURLStorage {
+#pragma mark - Private
+
+- (SDStateImageURLDictionary *)sd_imageURLStorage {
```

Comment:
Rename this to `sd_imageURLStorage` because category method is actually in the class method list. So this may cause name conflict. Seems not current, but have this possibility :)